geospaas_processing/downloaders.py

```diff
@@ -317,11 +311,21 @@ def download_dataset(self, dataset):
                 except (FileNotFoundError, IsADirectoryError) as error:
                     raise DownloadError(
                         f"Could not write the dowloaded file to {error.filename}") from error
+                else:
+                    if downloaded:
+                        LOGGER.info("Successfully downloaded dataset %d to %s",
+                                    dataset.pk, file_name)
+                    else:
+                        LOGGER.debug("Dataset %d is already present at %s", dataset.pk, file_name)
+                    return file_name
         raise DownloadError(f"Did not manage to download dataset {dataset.pk}")
 
     def download(self):
-        """Attempt to download all datasets matching the criteria"""
+        """Attempt to download all datasets matching the criteria."self.d_directory" attribute
+        should contain a pattern for the destination directory which is readable by 'strftime'. """
```

Comment:
The part about the pattern should be in the docstring of the constructor, were it is set.

Reply:
Done in the next commit